docs/run/run-sessions.mdx

```diff
@@ -108,11 +108,29 @@ with Session(service=service, backend=backend, max_time="25m"):
   ...   
 ```
 
-There is also an interactive timeout value (ITTL, or interactive time to live) that cannot be configured. For premium accounts, the ITTL is 5 min, and for open plan accounts, the ITTL is 2 sec. If no session jobs are queued within that window, the session is temporarily deactivated. For more details about session length and timeout, see [How long a session stays active](sessions#active).
+There is also an interactive timeout value (ITTL, or interactive time to live) that cannot be configured. For premium accounts, the ITTL is 1 min, and for open plan accounts, the ITTL is 2 sec. If no session jobs are queued within that window, the session is temporarily deactivated. For more details about session length and timeout, see [How long a session stays active](sessions#active).
 
 <span id="close"></span>
 ## Close a session 
 
+### For Qiskit 12.3 or later releases
+
+When the session context manager is exited, the session is put into `Not accepting new jobs` status.  This means that the session will finish processing all running or queued jobs until the maximum timeout value is reached.  After all jobs are completed, the session is immediately closed. This allows the
```

Comment:
> into `Not accepting new jobs` status

Let's stick with the statuses returned by `session.status()` defined [here](https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/session.py#L200-L206). These should match what IQP displays.

Reply:
Cloud API doesn't use those same status values does it? Maybe doesn't matter if qiskit is providing the translation from the API for both cloud and IQP channels.